dist/index.html

```diff
@@ -3,11 +3,11 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Tidal Migrations API</title>
+  <title>Tidal Migrations API <img src="tidal_icon.png" alt="Tidal Migrations logo"> </title>
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Source+Code+Pro:300,600|Titillium+Web:400,600,700" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="./swagger-ui.css" >
-  <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
-  <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
+  <link rel="icon" type="image/png" href="tidal_icon.png" sizes="32x32" />
```

Comment:
I think we actually need 2 different sized png files to support both 32x32 and 16x16 sized favicons.

More info here - https://stackoverflow.com/questions/4014823/does-a-favicon-have-to-be-32x32-or-16x16
This might be useful - http://www.favicomatic.com/
I haven't read that post or used that tool before, hopefully it's helpful.